dali/dfu/dfurun.cpp

```diff
@@ -1285,6 +1285,19 @@ class CDFUengine: public CInterface, implements IDFUengine
                     // keys default wrap for copy
                     if (destination->getWrap()||(iskey&&(cmd==DFUcmd_copy)))
                         destination->setNumPartsOverride(srcFile->numParts());
+#ifdef _CONTAINERIZED
+                    else
```

Comment:
minor: I know it is moving existing code, but 

```
else if (isContainerized())
```
is preferred since it means the code is compiled as part of the bare-metal build.  There may also be requirements down the line to support a similar mode in bare-metal.  (Really the test is for non-local storage rather than strictly containerized).

Reply:
Changed